source/_docs/automation.markdown

```diff
@@ -19,26 +19,26 @@ Before you can go ahead and create your own automations, it's important to learn
 
 The example consists of three different parts: a [trigger](/docs/automation/trigger/), a [condition](/docs/automation/condition/) and an [action](/docs/automation/action/).
 
-The first line is the **trigger** of the automation rule. Triggers describe events that should trigger the automation rule. In this case, it is a person arriving home, which can be observed in Home Assistant by observing the state of Paulus changing from 'not_home' to 'home'.
+The first line is the **trigger** of the automation rule. Triggers describe events that should start execution of the automation rule. In this case, it is a person arriving home, which can be observed in Home Assistant by Paulus' **state** going from 'not_home' to 'home'.
```

Comment:
> Thanks for the PR @50m3rer0 
> 
> I've reviewed it and left a few comments. Could you please take a look? Thanks

Hi Frenck. Thanks for getting back to me and commenting on the PR. I'm fairly new to HA and Git, i.e., what I don't know (for both) is way more than what I do know. Regarding your comments:

1. After seeing the Hacktoberfest post I felt encouraged to participate at the level I can: looking at documentation
2. Because of my noob condition, I'm not editing technical aspects, but only suggesting changes that would make it easier to follow the text
3. I'm happy to make changes to the PR I sent, but I'm not sure how to do that ([this post ](https://community.home-assistant.io/t/editing-the-documentation-and-creating-a-pull-request-on-github/9573/4)does not talk about it, but I'll research to try to figure it out)

This said, I'll now address each of your comments. 

Reply:
Go to the "File changed" tab above, click on the `...` menu on the file you want to adjust, select `Edit file`; and make the adjustment you'd like to make.